mobile/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -458,7 +458,7 @@
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 6;
```

Comment:
Not sure I follow here but I am used to version numbers of anything to always be going higher. 😀😀 please shed more light here?

Reply:
This PR was originally meant to update the version codes and create a release. But review took longer than expected. Ended up using it to resolve the error message .

Reply:
This is a build number not a version number.
For iOS, build numbers can be reset for individual version numbers.
For android, build number (version codes) have to be incremented irrespective of the version number.